### ISSUE TITLE
delinking sessions from meetings

### ISSUE THREAD

**Issue by anantshri** [null-open-security-community/swachalit#41]
Current Status:
At this point only when their is a session scheduled in a chapter meeting a person can add content in portal.

Proposed Status:
1. Any logged in User is allowed to create sessions. Not chapter meet independent sessions.
2. Sessions allow user to add description, slides / pdf, video.
3. The session should have some sort of moderation option.
4. Once approved session should be visible in search etc.
5. Probable sessions could be audio video sessions or whitepaper

**Comment by abhisek** [comment]
As I understand, this is required for our goal of supporting user driven content generation? So the feature need not be mapped or related to current `EventSession`. We can create another entity that allows a user to publish content with

- Name
- Description
- Suitable Event Type (for consideration in meet)
- Attachment(s)

This content, post moderation will show up on the website along with being `searchable`.

Is this a correct understanding of the requirement @anantshri ?

**Comment by anantshri** [comment]
Yes that is correct we also need to define the moderation flow who will see the moderation queue how to approve it etc.
once approved it should be visible in the users profile also as a session or submitted content etc

**Comment by abhisek** [comment]
Can you model this like HasGeek does? Allow users to create arbitrary session proposals listed independently along with ability to like/dislike. Any chapter lead, can create a EventSession from these sessions after having conversation with the proposer.

What do you think?
